model/date/core/package.json

```diff
@@ -19,6 +19,8 @@
   "dependencies": {
     "@cozemble/lang-util": "workspace:*",
     "@cozemble/model-core": "workspace:*",
-    "date-fns": "^2.29.3"
+    "date-fns": "^2.29.3",
+    "dayjs": "~1.11.7",
+    "moment": "^2.29.4"
```

Comment:
Sam, do we really need all three date packages?

Reply:
Updated, remove `date-fn` and `moment`